### PR TITLE
Use logger instead of print

### PR DESCRIPTION
**Pull request by nathanleiby**
- Converts print statements to use `logger.info` instead. https://github.com/Clever/ebs-snapshots/commit/94e63f190419c4c0653bec5f74c2b2293d4a3028
- Also adds pylint, to more rigorously check for errors https://github.com/Clever/ebs-snapshots/commit/ac2cec9974574b777f349872b0c19d0af6c7a492

### CODE REVIEW

File: ebs_snapshots/snapshot_manager.py [hunk 1]
```diff
@@ -67,10 +68,10 @@ def _ensure_snapshot(connection, volume, interval, name):
     :returns: None
     """
     if interval not in VALID_INTERVALS:
-        print kayvee.formatLog("ebs-snapshots", "warning", "invalid snapshotting interval", {
+        logging.info(kayvee.formatLog("ebs-snapshots", "warning", "invalid snapshotting interval", {
```
Comment (mohit) [inline ebs_snapshots/snapshot_manager.py]: any reason why we are not matching the log level to the one in the actual log line?

**Comment by mohit** [comment]
awesome! except not sure why we should use info everywhere.

File: ebs_snapshots/snapshot_manager.py (hunk 1 above)
Comment (nathanleiby) [inline ebs_snapshots/snapshot_manager.py]: Will make it match. Ideally we'll someday say `kayvee.<level>` and remove the formatLog too

**Comment by nathanleiby** [comment]
@mohit updated to match log level - 6d52fcb

**Pull request merged**